### ISSUE TITLE
[AccSpot] Failed Accessibility Error - Calculator.exe (Calculator.exe 10.1903.21.0) - Control type:UIA_HyperlinkControlType, Test:MissingProperty Error:UIA_NameProperty: Error - MissingProperty UIA_NameProperty , Priority - 0 , ElementControlType - UIA_Hy

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [Microsoft/calculator#487]
A team member will update this issue with more details soon.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
